src/pages/dashboard/_CDCComparisonContainer.js

```diff
@@ -49,25 +49,32 @@ export default function CDCComparisonContainer() {
     }))
     .reduce((acc, val) => acc.concat(val), [])
     .sort(sortChronologically)
-  const cdcCumulativeTotal = sum(cdcData, d => d.value)
-  const cumulativeTotal = sum(data, d => d.value)
+  // const cdcCumulativeTotal = sum(cdcData, d => d.value)
+  // const cumulativeTotal = sum(data, d => d.value)
   const dailyMax = max(data, d => d.value)
   return (
     <section>
       <div>
-        <h4>Differences between this data and the CDC data</h4>
         <p>
-          As of today, the C.D.C. has tested {formatNumber(cdcCumulativeTotal)}{' '}
-          specimens, and we have tracked at least{' '}
-          {formatNumber(cumulativeTotal)} tests administered across the country.
-          It is important to note that testing numbers are likely an undercount
-          because of the lack of universal testing and state reporting, and that
-          multiple specimen tests can be conducted from a single person’s
-          sample.
+          As of today, the COVID Tracking Project has identified more than 2.3
```

Comment:
I like having the calculated number in here.  We could accomplish something similar using `d3-format`

`d3.format('.2s')(cumulativeTotal).replace('M',' million')`